opentreemap/opentreemap/settings/default_settings.py

```diff
@@ -357,7 +357,17 @@
 # STORAGE_UNITS is the unit the value will be stored/computed as,
 # if different from DISPLAY_DEFAULTS
 #
-STORAGE_UNITS = {}
+STORAGE_UNITS = {
+    'bioswale': {
+        'drainage_area': 'sq_m'
+    },
+    'rainGarden': {
+        'drainage_area': 'sq_m'
+    },
+    'greenInfrastructure': {
+        'area': 'sq_m'
+    }
+}
```

Comment:
Polygon coordinates are in roughly meters, using map projection 3857 for use with standard tiling systems. The PostGIS that calculates area calculates sq meters because the points are in meters, which makes perfect sense. So I'd definitely prefer to keep greenInfrastructure.area natively in sq_m, since that's closest to what's really in the db.

I can see an argument for changing drainage_area, though.

Given that this is a 1-point task, I'm not sure it's worth the work. How strongly do you feel about it?